### ISSUE TITLE
Add language switcher

### ISSUE THREAD

**Issue by shivenbhatt1** [onaio/rdt-standard#278]
Add a language switcher (spinner) in the drawer menu with English and Bahasa as choices:

- [x] Selecting English should set the Locale `SharedPreferences` value to `en` 
- [x] Selecting Bahasa should set the Locale `SharedPreferences` value to `id`
- [x] Update the Locale from `SharedPreferences` when starting Activities

**Comment by pld** [comment]
does this depend on https://github.com/onaio/rdt-standard/issues/256 ?

**Comment by shivenbhatt1** [comment]
@pld Vincent asked me to put in a new issue for this

**Comment by pld** [comment]
> @pld Vincent asked me to put in a new issue for this

yea makes sense, trying to understand dependencies, since we'd have to bump that issue up in priority if it's a dependency on this

**Comment by vincent-karuri** [comment]
It does depend on #256. Otherwise, simply adding a language switcher won't have an effect on native forms without MLS activated.

**Comment by pld** [comment]
So we either have to add or remove both of those to this milestone, I'm going to remove this for now and we can reassess post Thursday's preview release

**Comment by pld** [comment]
oh never mind, I didn't realize 256 was already in this milestone, ignore my previous comment then

**Comment by vincent-karuri** [comment]
@rowo could we please get a simple design for the language spinner?

CC @syedowaisali90

**Comment by pld** [comment]
@ndegwamartin is there s/t for this in another app that we can reuse?

**Comment by rowo** [comment]
Here's is the language selector from BA. I think there's a version of this in mcard and WCARO also. Basically just a dialog that comes up if you hit that "world" button. Languages should be shown in that language's text.

![Menu-Stock](https://user-images.githubusercontent.com/1584163/91343787-08b93200-e7ab-11ea-9712-1b15bb0f3d17.png)

<img width="657" alt="Screen Shot 2020-08-26 at 2 48 36 PM" src="https://user-images.githubusercontent.com/1584163/91343986-4ae27380-e7ab-11ea-8d82-b918764aad73.png">
